Makefile.am

```diff
@@ -0,0 +1,39 @@
+AUTOMAKE_OPTIONS = subdir-objects
+
+bin_PROGRAMS = comskip
+if BUILD_GUI
+    bin_PROGRAMS += comskip-gui
+endif
+check_PROGRAMS = votest
+
+base_sources = comskip.c comskip.h mpeg2dec.c platform.c platform.h video_out_dx.c vo.h \
+    ccextratorwin/608.c ccextratorwin/608.h \
+    ccextratorwin/ccextractor.c ccextratorwin/ccextractor.h \
+    ccextratorwin/encoding.c \
+    ccextratorwin/general_loop.c \
+    ccextratorwin/myth.c
+
+comskip_SOURCES = $(base_sources)
+comskip_gui_SOURCES = $(base_sources) video_out_sdl.c
+votest_SOURCES = video_out_dx.c video_out_sdl.c
+
+DEFINES = -DPROCESS_CC
+if ENABLE_DONATOR
+    DEFINES += -DDONATOR
+endif
```

Comment:
I've reverted this to the old default behavior; ./configure --disable-donator can be used to make a public build.